src/index.html

```diff
@@ -17,16 +17,38 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="search__container">
```

Comment:
Use semantic tags where possible. For example, the search container could be a <section> or <header> depending on its context within the page.